bundle/manifests/3scale-operator.clusterserviceversion.yaml

```diff
@@ -471,15 +486,27 @@ spec:
                 ports:
                 - containerPort: 8080
                   name: metrics
+                - containerPort: 9443
+                  name: webhook-server
+                  protocol: TCP
                 resources:
                   limits:
                     cpu: 100m
                     memory: 300Mi
                   requests:
                     cpu: 100m
                     memory: 300Mi
+                volumeMounts:
+                - mountPath: /tmp/k8s-webhook-server/serving-certs
+                  name: cert
+                  readOnly: true
               serviceAccountName: 3scale-operator
               terminationGracePeriodSeconds: 10
+              volumes:
+              - name: cert
+                secret:
+                  defaultMode: 420
+                  secretName: webhook-server-cert
       permissions:
```

Comment:
Expecting something similar to the following to be generate and added to the CSV but it is not for some reason:

```
  webhookdefinitions:
  - admissionReviewVersions:
    - v1beta1
    - v1beta2
    containerPort: 443
    conversionCRDs:
    - products.capabilities.3scale.net
    deploymentName: threescale-operator-controller-manager-v2
    generateName: cproducts.capabilities.3scale.net
    sideEffects: None
    targetPort: 9443
    type: ConversionWebhook
    webhookPath: /convert
```